addons/web_tour/static/src/js/running_tour_action_helper.js

```diff
@@ -28,6 +28,9 @@ var RunningTourActionHelper = core.Class.extend({
     text: function (text, element) {
         this._text(this._get_action_values(element), text);
     },
+    text_blur: function (text, element) {
+        this._text_blur(this._get_action_values(element), text);
+    },
```

Comment:
I'll extract the web_tour part of this commit now and check with framework team to merge. The reset can thus be squashed.